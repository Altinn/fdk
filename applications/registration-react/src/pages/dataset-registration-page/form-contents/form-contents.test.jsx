import React from 'react';
import { shallow } from 'enzyme';
import {
  FormContentsComponent,
  renderStandard
} from './form-contents.component';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {};
  wrapper = shallow(<FormContentsComponent {...defaultProps} />);
});

test('should render FormContentsComponent, { renderS correctly', () => {
  expect(wrapper).toMatchSnapshot();
});

test('should render renderStandard correctly', () => {
  wrapper = shallow(renderStandard(defaultProps));
  expect(wrapper).toMatchSnapshot();
});
