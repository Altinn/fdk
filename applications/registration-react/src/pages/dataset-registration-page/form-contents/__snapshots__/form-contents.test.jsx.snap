// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormContentsComponent, { renderS correctly 1`] = `
<form>
  <div
    className="form-group"
  >
    <div
      className="mt-4"
    >
      <div
        className="form-group"
      >
        <withStateHandlers(Helptext)
          term="Dataset_conformsTo"
          title="Standard"
        />
        <FieldArray
          component={[Function]}
          linkLabel="Lenke"
          name="conformsTo"
          titleLabel="Tittel"
        />
      </div>
    </div>
  </div>
  <div
    className="form-group"
  >
    <withStateHandlers(Helptext)
      term="Dataset_hasQualityAnnotation_relevance"
      title="Relevans"
    />
    <Field
      component={[Function]}
      label="Relevans"
      name="hasRelevanceAnnotation.hasBody.nb"
    />
  </div>
  <div
    className="form-group"
  >
    <withStateHandlers(Helptext)
      term="Dataset_hasQualityAnnotation_completeness"
      title="Kompletthet"
    />
    <Field
      component={[Function]}
      label="Kompletthet"
      name="hasCompletenessAnnotation.hasBody.nb"
    />
  </div>
  <div
    className="form-group"
  >
    <withStateHandlers(Helptext)
      term="Dataset_hasQualityAnnotation_accuracy"
      title="Nøyaktighet"
    />
    <Field
      component={[Function]}
      label="Nøyaktighet"
      name="hasAccuracyAnnotation.hasBody.nb"
    />
  </div>
  <div
    className="form-group"
  >
    <withStateHandlers(Helptext)
      term="Dataset_hasQualityAnnotation_availability"
      title="Tilgjengelighet"
    />
    <Field
      component={[Function]}
      label="Tilgjengelighet"
      name="hasAvailabilityAnnotation.hasBody.nb"
    />
  </div>
</form>
`;

exports[`should render renderStandard correctly 1`] = `<div />`;
